cron/photos.py

```diff
@@ -97,6 +125,7 @@ def crawl_photos():
     data_bucket = storage_client.bucket("epschedule-data")
     schedule_blob = data_bucket.blob("schedules.json")
     schedules = json.loads(schedule_blob.download_as_string())
+    photoids{} = read_csv_as_dict()
 
     for username in schedules:
         student_schedule = schedules[username]

```

Comment:
if `photo_ids` is not `None` you'll need to get `sid` for each user and look up the url in `photo_ids`. Then call your new API instead of calling `download_photo`

Reply:
should i take out the logic using the username? i.e. for username in schedules...

Reply:
no, that's fine. you'll just need to get `sid` for each username since that will be your key to `photo_ids`.